### ISSUE TITLE
[Accessibility][TextField]: Need support to either add aria-describedby or an ability to hide "description"

### ISSUE THREAD

**Issue by vivekjilla** [OfficeDev/office-ui-fabric-react#1081]
#### Currently TextField Component supports aria-describedby using a prop called 'description'. We can provide some description and the component itself creates a div element and links it to the TextField using aria-describedby attribute.

But the problem with this is, we don't have much control over styling of the description.

e.g.: Most of the times we want to add aria-describedby for screen reader purposes only and we don't want to show the description on the page to normal users. In this case we would like to hide html element pointed by aria-describedby. 

So we need support to either pass aria-describedby so that we can control how it should behave on our end, or an option to hide description.

**Comment by mdahamiwal** [comment]
Have you tried adding `aria-describedby={ }` directly? It should work if you have your own description else where in the Dom.

**Comment by vivekjilla** [comment]
Yes. I tried adding aria-describedby directly. It is not creating that prop in the generated html.
